e2e/playwright-local.config.js

```diff
@@ -15,7 +15,7 @@ const config = {
         env: {
             NODE_ENV: 'test'
```

Comment:
@ozyx is this necessary if we're running with a new `npm run start:coverage` ?

Reply:
@scottbell @unlikelyzero :
This should not be necessary anymore since we were just using the envvar to control when webpack HMR launched (also coverage). We should also remove the `cross-env` logic from the other configs

Reply:
@scott where is HMR configured?

Reply:
@unlikelyzero in [this block](https://github.com/nasa/openmct/pull/5797/files#diff-52cf7951fec7ee5c8db1888ab977bc94c938511c211beecca96a1ebb64a4dafaR38). Per the [webpack docs](https://webpack.js.org/configuration/dev-server/#devserverhot):

> Since webpack-dev-server v4, HMR is enabled by default. It automatically applies [webpack.HotModuleReplacementPlugin](https://webpack.js.org/plugins/hot-module-replacement-plugin/) which is required to enable HMR. So you don't have to add this plugin to your webpack.config.js when hot is set to true in config or via the CLI option --hot. See the [HMR concepts page](https://webpack.js.org/concepts/hot-module-replacement/) for more information.